test/clojure_commons/jwt_test.clj

```diff
@@ -77,21 +76,21 @@
 (defn- build-custom-assertion
   [validity-window-end {:keys [user email given-name family-name common-name]}]
   (let [now (time/now)]
-    {:exp                                 (time/plus now (time/seconds validity-window-end))
-     :iat                                 now
-     :http://wso2.org/claims/enduser      (str "foo/" user)
-     :http://wso2.org/claims/emailaddress email
-     :http://wso2.org/claims/givenname    given-name
-     :http://wso2.org/claims/lastname     family-name
-     :http://wso2.org/claims/fullname     common-name}))
+    {:exp                               (time/plus now (time/seconds validity-window-end))
+     :iat                               now
+     :http:wso2.org.claims.enduser      (str "foo/" user)
+     :http:wso2.org.claims.emailaddress email
+     :http:wso2.org.claims.givenname    given-name
+     :http:wso2.org.claims.lastname     family-name
+     :http:wso2.org.claims.fullname     common-name}))
```

Comment:
Keywords in the previous format caused lint warnings. We don't get JWTs from WSO2 anymore, so this shouldn't be much of a problem for us.